content/id/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -0,0 +1,307 @@
+_container runtime_ yang terinstal dengan memindai sekumpulan soket domain Unix yang umum digunakan.
+Tabel berikut menunjukkan _container runtime_ dan lokasi soketnya:
+
+{{< table caption = "Container runtime dan lokasi soketnya" >}}
+| Runtime    | Path to Unix domain socket        |
+|------------|-----------------------------------|
+| Docker     | `/var/run/docker.sock`            |
+| containerd | `/run/containerd/containerd.sock` |
+| CRI-O      | `/var/run/crio/crio.sock`         |
+{{< /table >}}
+
+<br />
+Jika ditemukan Docker dan containerd secara bersamaan, Docker akan terpilih. Hal ini diperlukan
+karena Docker 18.09 dirilis dengan containerd dan keduanya dapat ditemukan meskipun kamu
+hanya menginstal Docker.
+Jika ditemukan selain dari kedua _container runtime_ tersebut, kubeadm akan berhenti dengan kegagalan.
+
+kubelet berintegrasi dengan Docker melalui implementasi CRI `dockershim` bawaannya.
+
+Lihat [container runtime](/docs/setup/production-environment/container-runtimes/)
+untuk informasi lebih lanjut.
+{{% /tab %}}
+{{% tab name="sistem operasi lainnya" %}}
+Secara bawaan, kubeadm menggunakan {{< glossary_tooltip term_id="docker" >}} sebagai _container runtime_.
+kubelet berintegrasi dengan Docker melalui implementasi CRI `dockershim` bawaannya.
+
+Lihat [container runtime](/docs/setup/production-environment/container-runtimes/)
+untuk informasi lebih lanjut.
+{{% /tab %}}
+{{< /tabs >}}
+
+
+## Menginstal kubeadm, kubelet, dan kubectl
+
+Kamu akan menginstal _package_ berikut pada semua mesinmu:
+
+* `kubeadm`: perkakas untuk mem-_bootstrap_ klaster.
+
+* `kubelet`: komponen yang berjalan pada seluruh mesin pada klaster
+    dan memiliki tugas seperti menjalankan Pod dan kontainer.
+
+* `kubectl`: perkakas untuk berinteraksi dengan klastermu.
+
+kubeadm **tidak akan** menginstal atau mengelola `kubelet` ataupun `kubectl` untukmu, jadi kamu harus memastikan
+keduanya memiliki versi yang cocok dengan _control plane_ Kubernetes yang akan kamu instal
+dengan kubeadm. Jika tidak, ada risiko _version skew_ yang dapat terjadi dan
+dapat berujung pada perangai yang bermasalah dan tidak terduga. Namun, _satu_ _version skew_ minor antara
+kubelet dan _control plane_ masih diperbolehkan, tetapi versi kubelet tidak boleh melebihi versi API
+Server. Sebagai contoh, kubelet yang berjalan pada versi 1.7.0 akan kompatibel dengan API Server versi 1.8.0, tetapi tidak sebaliknya.
+
+Untuk informasi mengenai instalasi `kubectl`, lihat [Menginstal dan mengatur kubectl](/docs/tasks/tools/install-kubectl/).
+
+{{< warning >}}
+Instruksi ini membuat seluruh _package_ Kubernetes keluar dari _system upgrade_.
+Hal ini karena kubeadm dan Kubernetes membutuhkan
+[perhatian khusus untuk pembaharuan](/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/).
+{{</ warning >}}
+
+Untuk informasi lebih lanjut mengenai _version skew_, lihat:
+
+* Kubernetes [version and version-skew policy](/docs/setup/release/version-skew-policy/)
```

Comment:
```suggestion
* Kubernetes [versi dan kebijakan _version-skew_](/docs/setup/release/version-skew-policy/)
```

Reply:
FIxed in 0b74a869bea33a4eb86e43a89b5284bcd244444f